forge-gui/res/cardsfolder/upcoming/chaotic_transformation.txt

```diff
@@ -0,0 +1,12 @@
+Name:Chaotic Transformation
+ManaCost:5 R
+Types:Sorcery
+A:SP$ Pump | ValidTgts$ Artifact | TgtPrompt$ Select target artifact | TargetMin$ 0 | TargetMax$ 1 | SubAbility$ DestroyCreature | AILogic$ Destroy | IsCurse$ True | SpellDescription$ Destroy target artifact, target creature, target enchantment, and target land. | StackDescription$ None
+SVar:DestroyCreature:DB$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | TargetMin$ 0 | TargetMax$ 1 | AILogic$ Destroy | IsCurse$ True | SubAbility$ DestroyEnch | StackDescription$ None
+SVar:DestroyEnch:DB$ Pump | ValidTgts$ Enchantment | TgtPrompt$ Select target enchantment | TargetMin$ 0 | TargetMax$ 1 | AILogic$ Destroy | IsCurse$ True | SubAbility$ DestroyPlaneswalker | StackDescription$ None
+SVar:DestroyPlaneswalker:DB$ Pump | ValidTgts$ Planeswalker | TgtPrompt$ Select target planeswalker | TargetMin$ 0 | TargetMax$ 1 | AILogic$ Destroy | IsCurse$ True | SubAbility$ DestroyLand | StackDescription$ None
+SVar:DestroyLand:DB$ Pump | ValidTgts$ Land | TgtPrompt$ Select target land | TargetMin$ 0 | TargetMax$ 1 | AILogic$ Destroy | IsCurse$ True | SubAbility$ DBDestroy | StackDescription$ None
+SVar:DBDestroy:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | Defined$ Targeted | SubAbility$ DBRepeat | RememberLKI$ True
+SVar:DBRepeat:DB$ RepeatEach | DefinedCards$ DirectRemembered | RepeatSubAbility$ DBRevealUntilShare
```

Comment:
Add `ClearRemembered$` to avoid collision and try again, maybe that will make it work